CODEOWNERS

```diff
@@ -0,0 +1,30 @@
+# https://help.github.com/en/articles/about-code-owners
+# Documentation: https://github.com/hmcts/cnp-flux-config/blob/master/CODEOWNERS.md
+
+CODEOWNERS @hmcts/platform-engineering @hmcts/devops
+
+# AAT applications
+## environment rules
+k8s/aat/ @hmcts/platform-engineering @hmcts/devops
+
+## application rules
+
+### RPE
+k8s/aat/common/rpe/namespace.yaml @hmcts/platform-engineering
```

Comment:
I thought we should be able to do k8s/**/common/rpe/namespace.yaml @hmcts/platform-engineering. so that we can avoid duplication.

Reply:
not sure if we really get any benefit with that and we lose explicitness. don't really mind though

Reply:
I think it will be good to reduce duplication. but it would mean k8s/sandbox will also have same approvals . But worried about this file growing too big.

Reply:
let's give it a go have pushed a commit